app/views/studies/show.html.erb

```diff
@@ -340,37 +340,40 @@
 </div>
 
 <div class="panel panel-success" id="study-shares-panel">
-	<div class="panel-heading">
-		<div class="panel-title">
-			<h4><%= link_to "Sharing <span class='badge' id='share-count'>#{@study.study_shares.visible.size}</span> <span class='fas fa-chevron-down toggle-glyph'></span>".html_safe, '#study-shares', class: 'text-success', 'data-toggle' => 'collapse' %></h4>
-		</div>
-	</div>
+  <div class="panel-heading">
+    <div class="panel-title">
+      <h4><%= link_to "Sharing <span class='badge' id='share-count'>#{@study.study_shares.visible.size}</span> <span class='fas fa-chevron-down toggle-glyph'></span>".html_safe, '#study-shares', class: 'text-success', 'data-toggle' => 'collapse' %></h4>
+    </div>
+  </div>
 
-	<div class="panel-collapse collapse in" id="study-shares">
-		<div class="panel-body table-responsive">
-			<table class="table table-striped">
-				<thead>
-				<tr>
-					<th>Email</th>
-					<th>Permission</th>
-					<th>Date</th>
-				</tr>
-				</thead>
-				<tbody>
-				<% @study.study_shares.each do |study_share| %>
+  <div class="panel-collapse collapse in" id="study-shares">
+    <div class="panel-body table-responsive">
+      <table class="table table-striped">
+        <thead>
+        <tr>
+          <th>Email</th>
+          <th>Permission</th>
+          <th>Date</th>
+        </tr>
+        </thead>
+        <tbody>
+        <% @study.study_shares.each do |study_share| %>
           <%#=% Don't show entry for read-only service account %>
           <% if study_share.show_share? %>
-					<tr id="study-share-<%= study_share.email.gsub(/[@.]/, '-') %>">
-						<td class="share-email"><%= study_share.email %></td>
-						<td class="share-permission"><%= study_share.permission %></td>
-						<td><%= study_share.updated_at.to_s(:long) %></td>
-					</tr>
+          <tr id="study-share-<%= study_share.email.gsub(/[@.]/, '-') %>">
+            <td class="share-email"><%= study_share.email %></td>
+            <td class="share-permission">
+              <%= study_share.permission %><br />
+              <span class="help-block share-description"><%= StudyShare::PERMISSION_DESCRIPTION_MAP[study_share.permission] %></span>
```

Comment:
Nice touch.  

Note to self: this resolves to to "Edit, View, Reviewer" via [here](https://github.com/broadinstitute/single_cell_portal_core/blob/939f7e397a7902ba0fe373680677a9e594575e63/app/models/study_share.rb#L40) and [here](https://github.com/broadinstitute/single_cell_portal_core/blob/939f7e397a7902ba0fe373680677a9e594575e63/app/models/study_share.rb#L29).